### PR TITLE
Project Movies - Anne-Sophie

### PR DESCRIPTION
**Pull request by annesophiegendron**
Link to my deployed project:
https://elated-ride-645ab8.netlify.com/

My reflections on how this week's project turned out:
This project was really fun! I think I could have spared some time using a function to choose the category instead of duplicating the code into different js files for each page (where the only change was only a keyword in the API URL)
Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Comment by annesophiegendron** [comment]
Hi! I can see it when I open it 🤔

File: code/src/pages/MovieDetails.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+import 'pages/movieDetails.css';
+
+export const MovieDetails = () => {
+  const api_key = "67a5c607e91b1a60e64b4e131b88956b";
```
Comment (ericaglimsholt) [inline code/src/pages/MovieDetails.js:7]: Good idea with seperate the api key to a variable

File: code/src/pages/NowPlaying.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import 'pages/popularList.css'
+
+export const NowPlaying = () => {
+  const api_key = "67a5c607e91b1a60e64b4e131b88956b";
+  const [movies, setMovies] = useState([]);
+
+  // https://api.themoviedb.org/3/movie/now_playing?api_key=67a5c607e91b1a60e64b4e131b88956b&language=en-US&page=1
```
Comment (ericaglimsholt) [inline code/src/pages/NowPlaying.js:9]:
```suggestion
```

File: code/src/pages/PopularList.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import 'pages/popularList.css'
+
+export const PopularList = () => {
+  const api_key = "67a5c607e91b1a60e64b4e131b88956b";
+  const [movies, setMovies] = useState([]);
+
+  // https://api.themoviedb.org/3/movie/popular?api_key=67a5c607e91b1a60e64b4e131b88956b
```
Comment (ericaglimsholt) [inline code/src/pages/PopularList.js:9]:
```suggestion
```

File: code/src/pages/PopularList.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      })
+  }, [])
+
+
+  return (
+    <div className="main-wrapper">
+      <section className="main-content">
+        {movies.map((movie) => (
+          <Link key={movie.id} to={`/movies/${movie.id}`}>
+            <img src={`https://image.tmdb.org/t/p/w342${movie.poster_path}`} alt={movie.id}></img>
```
Comment (ericaglimsholt) [inline code/src/pages/PopularList.js:26]: Maybe the id name would suit as an image description?

**Pull request closed without merging**